server/web/services/submission_service.py

```diff
@@ -4,11 +4,14 @@
 
 from .. import database
 from ..models.submission_model import Submission
+from ..models.environment_model import Environment
+from ..models.game_model import Game
 
-def evaluate_agent(agent):
-    env = gym.make("FrozenLake-v0")
+def evaluate_agent(agent, game_id):
+    env_name = Environment.from_dict(list(database.find_env_name_by_game_id(game_id, show_fields=["gym_env"]))[0]).gym_env
```

Comment:
- Move this line to a method in environment_service.py called find_env_by_game.
- Change the name of the method find_env_name_by_game_id to find_env_by_game, and it should return env id and env name (and not game id).